conf/riff-raff.yaml

```diff
@@ -0,0 +1,21 @@
+stacks: [membership]
+regions: [eu-west-1]
+deployments:
+  cfn:
+    type: cloud-formation
+    app: contributions-frontend
+    parameters:
+      prependStackToCloudFormationStackName: false
+      cloudFormationStackName: contributions-frontend
+      templatePath: cfn.yaml
+  contributions-frontend-ami:
+    type: ami-cloudformation-parameter
+    app: contributions-frontend
```

Comment:
It's interesting how often this is repeated, feels like a bit of a smell cc @sihil